velox/dwio/parquet/tests/reader/ParquetReaderBenchmark.cpp

```diff
@@ -136,6 +152,11 @@ class ParquetReaderBenchmark {
     return rowReader;
   }
 
+  // This method is the place where we do the read opeartion.
+  // scanSpec contains the setting of filter and null-filter.
+  // e.g. filter 30% filter 70% of data and 30% remain.
```

Comment:
// e.g. filterRate = 30% (filterRateX100 = 30) means it would filter out 70% of rows and 30% remain.